src/Plugin/Field/FieldFormatter/CaptureUtilityMapFormatter.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Drupal\web_page_archive\Plugin\Field\FieldFormatter;
+
+use Drupal\Component\Utility\Html;
+use Drupal\Core\Field\FieldItemInterface;
+use Drupal\Core\Field\FieldItemListInterface;
+use Drupal\Core\Field\FormatterBase;
+
+/**
+ * Plugin implementation of the 'web_page_archive_capture_formatter' formatter.
+ *
+ * @FieldFormatter(
+ *   id = "web_page_archive_capture_utility_map_formatter",
+ *   label = @Translation("Web page archive capture utility map formatter"),
+ *   field_types = {
+ *     "map"
+ *   }
+ * )
+ */
+class CaptureUtilityMapFormatter extends FormatterBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function viewElements(FieldItemListInterface $items, $langcode) {
+    $elements = [];
+    foreach ($items as $delta => $item) {
+      $elements[$delta] = ['#markup' => $this->viewValue($item)];
+    }
+
+    return $elements;
+  }
+
+  /**
+   * Generate the output appropriate for one field item.
+   *
+   * @param \Drupal\Core\Field\FieldItemInterface $item
+   *   One field item.
+   *
+   * @return string
+   *   The textual output generated.
+   */
+  protected function viewValue(FieldItemInterface $item) {
+    return nl2br(Html::escape(implode("\n", array_values($item->toArray()))));
```

Comment:
@pobtastic, admittedly this is really just adapted from what Drupal Console generates, but just adapted since the item is an array instead of a string. 
- see: https://github.com/hechoendrupal/drupal-console/blob/42eb98db8e3c9955c7d1af0d13baa802e584f983/templates/module/src/Plugin/Field/FieldFormatter/fieldformatter.php.twig#L91

I'd be happy to adjust, but just curious if you could provide an example of what you're referring to so I can make sure it's done properly.